### PR TITLE
Move the DynamoDB driver to github.com/seedboxtech/eh-dynamo

### PR DESCRIPTION
**Pull request by mcarriere**
# Summary
Move the DynamoDB driver to https://github.com/seedboxtech/eh-dynamo after a discussion with @maxekman

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ further defined and clarified by project maintainers.
 ## Enforcement
 
 Instances of abusive, harassing, or otherwise unacceptable behavior may be
-reported by contacting the project team at [INSERT EMAIL ADDRESS]. All
```
Comment (maxekman) [inline CODE_OF_CONDUCT.md]: Nice find :)

File: eventstore/dynamodb/eventstore.go [hunk 2]
```diff
@@ -1,373 +0,0 @@
-// Copyright (c) 2016 - The Event Horizon authors.
```
Comment (maxekman) [inline eventstore/dynamodb/eventstore.go]: Please consider keeping this copy header in the derived version, changing "The Event Horizon authors" to "The Event Horizon DynamoDB authors" and keeping an AUTHORS file in the root (with all current authors of the DynamoDB related changes in it).

**Comment by maxekman** [comment]
Would be nice to get these fixed before merging this:
- https://github.com/seedboxtech/eh-dynamo/issues/2
- https://github.com/seedboxtech/eh-dynamo/issues/3
- https://github.com/seedboxtech/eh-dynamo/issues/4
- https://github.com/seedboxtech/eh-dynamo/issues/5

File: eventstore/dynamodb/eventstore.go (hunk 2 above)
Comment (mcarriere) [reply eventstore/dynamodb/eventstore.go]: fixed here https://github.com/seedboxtech/eh-dynamo/commit/4ad7e11a39cbd22d0463d2fff271b3ee5a64cd9c

**Comment by mcarriere** [comment]
I fixed all the issues.

Tell me if there's something more I should do :)

**Comment by maxekman** [comment]
@mcarriere Would be nice if you could set a Github description for your repo also.

**Comment by mcarriere** [comment]
> @mcarriere Would be nice if you could set a Github description for your repo also.

done!

**Pull request merged**